flyteplugins/go/tasks/plugins/k8s/ray/ray.go

```diff
@@ -489,12 +489,18 @@ func (plugin rayJobResourceHandler) GetTaskPhase(ctx context.Context, pluginCont
 			return pluginsCore.PhaseInfoFailure(flyteerr.TaskFailedWithError, reason, info), nil
 		case rayv1alpha1.JobStatusSucceeded:
 			return pluginsCore.PhaseInfoSuccess(info), nil
-		case rayv1alpha1.JobStatusPending, rayv1alpha1.JobStatusRunning:
+		case rayv1alpha1.JobStatusPending, rayv1alpha1.JobStatusRunning, rayv1alpha1.JobStatusStopped:
```

Comment:
Stopped sounds weird to be running

Reply:
It's not currently being used in the operator... so I took a guess it's a temporary state (Succeeded/Failed are terminal)... hence why I marked it as Running. That said, I think it's ok for now to add it to the default handler of failing as an unknown/not handled state..